src/index.html

```diff
@@ -7,8 +7,78 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#" class="logo_link">
+        <img
+          src="images/logo.png"
+          alt="Moyo logo"
+          class="logo_img"
+        >
+      </a>
+      <nav class="moyo_nav">
+        <ul class="moyo_nav__list">
+          <li class="moyo_nav__item">
+            <a
+              href="#"
+              class="moyo_nav__link is-active">
```

Comment:
Should be the same formatting like for img above :
```
  class="moyo_nav__link is-active"
>
  Apple
</a>
```
Please remember for the future